test.js

```diff
@@ -452,7 +453,7 @@ test('alias - require previousId and userId', t => {
   })
 })
 
-test('e2e', async t => {
+test('e2e', retry(async t => {
```

Comment:
Because we call `t.fail()`, `async-retry` won't work here. Instead, I think we need to throw an error.